scrapi/base/__init__.py

```diff
@@ -110,26 +136,29 @@ def get_contributors(self, result):
         or creators in the OAI namespaces """
 
         contributors = result.xpath(
-            '//dc:contributor/node()', namespaces=self.NAMESPACES) or ['']
+            '//dc:contributor/node()',
+            namespaces=self.NAMESPACES
+        ) or []
         creators = result.xpath(
-            '//dc:creator/node()', namespaces=self.NAMESPACES) or ['']
+            '//dc:creator/node()',
+            namespaces=self.NAMESPACES
+        ) or []
 
         all_contributors = contributors + creators
 
         contributor_list = []
         for person in all_contributors:
-            if person:
```

Comment:
Since the list will just be empty if there are no contributors, no more need for the null check.